03-setup-mysql.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Sets up schemas in MySQL database for sample service.
+# Prerequisites:
+# - kubectl is installed and available on the PATH: https://kubernetes.io/docs/tasks/tools/
```

Comment:
`goose` should also be installed as a prereq

Reply:
no longer using goose